packages/sak-app/src/data/fpsakApi.ts

```diff
@@ -82,6 +82,7 @@ export const FpsakApiKeys = {
   PREVIEW_MESSAGE_TILBAKEKREVING: new RestKey<any, any>('PREVIEW_MESSAGE_TILBAKEKREVING'),
   PREVIEW_MESSAGE_FORMIDLING: new RestKey<any, ForhåndsvisMeldingParams>('PREVIEW_MESSAGE_FORMIDLING'),
   PREVIEW_MESSAGE_TILBAKEKREVING_HENLEGGELSE: new RestKey<any, any>('PREVIEW_MESSAGE_TILBAKEKREVING_HENLEGGELSE'),
+  HAR_IKKE_ADRESSE: new RestKey<boolean, any>('HAR_IKKE_ADRESSE'),
```

Comment:
Her bør det vel stå { saksnummer: string } i staden for any (sidan det er input til rest-kallet)

Reply:
Helt enig!